src/alda/repl/commands/ReplLoad.java

```diff
@@ -83,8 +86,8 @@ private void loadFile(String args, AldaServer server, StringBuffer history,
 
       if (!error) {
         // Check if we can continue overwrite.
-        System.out.println("This action will overwrite the current score. Continue?");
-        if (Util.promptWithChoices(reader, Arrays.asList("yes", "no")) == "yes"){
+        if (Util.uncheckedPromptWithChoices(reader, OVERWRITE_WARNING,
+                                            "yes", "no") == "yes"){
```

Comment:
oh yah this is my bad, you're definetly supposed to use `equals()`, I keep forgetting...